index.js

```diff
@@ -0,0 +1,15 @@
+let slideIndex = 0;
+let imgSrc = [
+  "/Assets/HeroImages/evermore-carousel-img.png",
+  "/Assets/HeroImages/fearless-carousel-img.png",
+  "/Assets/HeroImages/red-carousel-img.png"
+];
+
```

Comment:
You can add some comments to describe the function.

Reply:
Right, thanks for pointing that